app/helpers/password_resets_helper.rb

```diff
@@ -0,0 +1,24 @@
+module PasswordResetsHelper
+  def extract_user_from_reset_params
+    @user = User.find_by(email: params[:password_reset][:email].downcase)
+  end
+
+  def get_user_by_email
```

Comment:
it's not `get_user_by_email` call it `locate_user_by_email` some word apart from the CRUD actions

Reply:
ok